rcongui/src/components/MapManager/DraggableListItem.js

```diff
@@ -43,7 +46,7 @@ const DraggableListItem = ({ item, index, onRemove }) => {
     if (fullName.toLowerCase().endsWith("ger")) {
       labels.push("axis");
     }
-    if (fullName.search("night") !== -1) {
+    if (fullName.toLowerCase().search("night") !== -1) {
```

Comment:
ah, copy&paste :D We should probably use includes here as well.

Reply:
Much better, thanks! I am so JS ignorant.